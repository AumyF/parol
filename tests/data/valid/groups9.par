%start Start
%%
Start: ("x" ("a" | "b")) "y";
