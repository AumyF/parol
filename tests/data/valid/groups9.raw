%start Start
%title ""

%%

Start: "x" StartGroup "y";
StartGroup: "a";
StartGroup: "b";
