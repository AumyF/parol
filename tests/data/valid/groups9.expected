title: None
comment: None
start_symbol: Start
current_scanner: INITIAL
name: INITIAL;line_comments: [];block_comments: [];auto_newline_off: false;auto_ws_off: false;
Start: Alts(Alt((Alts(Alt(<0>T(x), (Alts(Alt(<0>T(a)) | Alt(<0>T(b))))))), <0>T(y)));
